src/crypto/algorithms/megolm.js

```diff
@@ -656,6 +665,7 @@ MegolmDecryption.prototype.onRoomKeyEvent = function(event) {
     this._olmDevice.addInboundGroupSession(
         content.room_id, senderKey, sessionId,
         content.session_key, event.getKeysClaimed(),
+        exportFormat,
```

Comment:
ok, for now I'm leaving keysClaimed unset.

keysProved is harder to do much about because it's inferred from the megolm session. In any case I think the constraint holds more-or-less as much as it ever did.